cmd/namespace.go

```diff
@@ -7,18 +7,52 @@ import (
 	"github.com/CircleCI-Public/circleci-cli/api"
 	"github.com/CircleCI-Public/circleci-cli/client"
 	"github.com/CircleCI-Public/circleci-cli/settings"
+	"github.com/manifoldco/promptui"
 	"github.com/spf13/cobra"
 )
 
 type namespaceOptions struct {
 	cfg  *settings.Config
 	cl   *client.Client
 	args []string
+
+	// Allows user to skip y/n confirm when creating a namespace
+	noPrompt bool
+	// This lets us pass in our own interface for testing
+	tty createNamespaceUserInterface
+	// Linked with --integration-testing flag for stubbing UI in gexec tests
+	integrationTesting bool
+}
+
+type createNamespaceUserInterface interface {
+	askUserToConfirm(message string) bool
+}
+
+type createNamespaceInteractiveUI struct{}
+
+func (ui createNamespaceInteractiveUI) askUserToConfirm(message string) bool {
+	prompt := promptui.Prompt{
+		Label:     message,
+		IsConfirm: true,
+	}
+
+	result, err := prompt.Run()
+	return err == nil && strings.ToLower(result) == "y"
```

Comment:
There's quite a lot of code here similar to https://github.com/CircleCI-Public/circleci-cli/pull/260 - is it easy to DRY it up a bit?

Reply:
I will create a follow up PR after this is merged

Reply:
#262